### PR TITLE
feat:support cypher parameter

### PR DESCRIPTION
**Pull request by xigongdaEricyang**
support param cypher, shouldn't merge until the nebula-go repo merged

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,7 +12,8 @@ nebula-httpd
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 .idea
+.vscode/**/*
 vendor/
-
+go.sum
```
Comment (veezhang) [inline .gitignore:17]: I think the `go.sum` should not be ignored. Why ignore it?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (veezhang) [inline .vscode/launch.json:1]: It is a personal configuration, no need to upload?

File: service/pool/pool.go [hunk 3]
```diff
@@ -100,7 +101,7 @@ func NewConnection(address string, port int, username string, password string) (
 								}
 							}
 						}()
-						response, err := connection.session.Execute(gql)
+						response, err := connection.session.ExecuteWithParameter(gql, request.ParamsMap)
```
Comment (xigongdaEricyang) [inline service/pool/pool.go:104]: Here need nebula-go pr approved to support this method

File: controllers/db.go [hunk 4]
```diff
@@ -26,7 +26,8 @@ type Request struct {
 }
 
 type ExecuteRequest struct {
-	Gql string `json:"gql"`
+	Gql       string             `json:"gql"`
+	ParamsMap common.ParamterMap `json:"paramsMap"`
```
Comment (nianiaJR) [inline controllers/db.go:30]: keep the name style same： `ParamterMap` =>(fix typo) `ParameterMap` => `ParamsMap`

Comment (xigongdaEricyang) [reply controllers/db.go:30]: ok， let me fix it

**Pull request closed without merging**